src/data-tutorials/users.tsx

```diff
@@ -2070,11 +2070,18 @@ const Users: User[] = [
     tags: ['javascript', 'react', 'beginner', 'celo'], 
   },
   {
+    title: 'Connect and interact with Celo using web3modal',
+    description: 'Alternative method of connecting to Celo using web3React',
+    preview: require('./showcase/advanced/connect-and-interact-with-celo-using-web3modal.png'),
```

Comment:
`./showcase/advanced/connect-and-interact-with-celo-using-webmodal.png`
change the url here

Reply:
This path seems to be correct. Could you please check again?

Reply:
If you check correctly you used web3modal but in image name its webmodal.png

Reply:
@bobeu